web/src/pages/contact.tsx

```diff
@@ -1,15 +1,96 @@
-import { Link } from 'gatsby';
+import { Box, Flex, Heading, Label, Link } from '@theme-ui/components';
 import React from 'react';
+import { SystemStyleObject } from 'theme-ui';
+import MapSvg from '../assets/map.svg';
+import { EmployeeList } from '../components/contact/exployee-list';
 import { Container } from '../components/container';
 import { Layout } from '../components/layouts';
 
+const styles: SystemStyleObject = {
+  '.image': {
+    width: '396px',
+    height: '328px',
+    right: '180px',
+    position: 'absolute',
+    top: '200px',
+  },
```

Comment:
This has multiple issues. Looks like we're position something in absolute; I always ask myself, do we need to do that? Is there **any** other way than position?

Height and width should be defined with some dynamic value (%,rem,em) and the top and right values are `>30px`. We should probably try to implement this another way.